### ISSUE TITLE
Typescript errors after adding plugin

### ISSUE THREAD

**Issue by melink14** [nguyenbatranvan/vite-multiple-assets#40]
The problems seem to be caused by the incompatible interfaces defined here:
https://github.com/nguyenbatranvan/vite-multiple-assets/blob/e10e7a717cc729d9da09e02a86f7c83be03d2715/packages/libs/src/types.d.ts#L102

I'm not sure if the old interface was meant to be deleted when the new one was added or why this hasn't been seen by others (perhaps typechecking is rare in vite or my configuration is stricter?)

Full error ouptut:
```
> tsc --noEmit

node_modules/vite-multiple-assets/dist/index.d.mts:113:2 - error TS2687: All declarations of 'options' must have identical modifiers.

113  options: IConfig;
     ~~~~~~~

node_modules/vite-multiple-assets/dist/index.d.mts:119:2 - error TS2717: Subsequent property declarations must have the same type.  Property 'assets' must be of type 'IAssets', but here has type 'string[]'.

119  assets: string[];
     ~~~~~~

  node_modules/vite-multiple-assets/dist/index.d.mts:112:2
    112  assets: IAssets;
         ~~~~~~
    'assets' was also declared here.

node_modules/vite-multiple-assets/dist/index.d.mts:120:2 - error TS2687: All declarations of 'options' must have identical modifiers.

120  options?: IConfig;
     ~~~~~~~

node_modules/vite-multiple-assets/dist/index.d.mts:120:2 - error TS2717: Subsequent property declarations must have the same type.  Property 'options' must be of type 'IConfig', but here has type 'IConfig | undefined'.

120  options?: IConfig;
     ~~~~~~~

  node_modules/vite-multiple-assets/dist/index.d.mts:113:2
    113  options: IConfig;
         ~~~~~~~
    'options' was also declared here.


Found 4 errors in the same file, starting at: node_modules/vite-multiple-assets/dist/index.d.mts:113
```

**Comment by nguyenbatranvan** [comment]
From what I understand, this indicates a naming conflict related to the type declaration! You can check out some resources or solutions on GitHub for reference.
- [https://github.com/microsoft/TypeScript/issues/26389](https://github.com/microsoft/TypeScript/issues/26389)
- [https://github.com/DefinitelyTyped/DefinitelyTyped/issues/15104](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/15104)

**Comment by nguyenbatranvan** [comment]
Let's review the solutions with the approaches mentioned above. I will soon release a version that removes the duplicate type definition of `IParameterViteServe`.

**Comment by nguyenbatranvan** [comment]
fixed `2.2.5`
